src/Ocelot/DownstreamUrlCreator/Middleware/DownstreamUrlCreatorMiddleware.cs

```diff
@@ -165,5 +166,10 @@ private static bool ServiceFabricRequest(IInternalConfiguration config, Downstre
         {
             return config.ServiceProviderConfiguration.Type?.ToLower() == "servicefabric" && downstreamRoute.UseServiceDiscovery;
         }
+
+        /// <summary>
+        /// Takes in a string value and transforms it, so it can be later used in a Regex constructor.
+        /// </summary>
+        private static string BuildRegexSafeValue(string input) => Regex.Escape(input);

```

Comment:
I find little purpose in this static helper. It takes a single parameter and merely delegates to the `Regex` helper.